.buildkite/ftr_configs.yml

```diff
@@ -1,6 +1,7 @@
 disabled:
   # TODO: Enable once RBAC timeline search strategy test updated
-  - x-pack/test/timeline/security_and_spaces/config_basic.ts
+  - x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint/configs/config.base.ts
+  - x-pack/test/security_solution_api_integration/test_suites/security_solution_endpoint_api_int/configs/config.base.ts
```

Comment:
Is it intended to list these base configs under this TODO comment? Or should they rather move down under the `# Base config files, only necessary to inform config finding script` comment?